reset_config: [-0.135214, 0.18, 0.1, -1.52432, 2.67237]
encoder_offsets: [0.0, 0.08, -0.025, -0.075, 0.0]
#encoder_offsets: [0.0, 0.0, 0.0, 0.0, 0.0]
